R/zchunk_L244.building_USA.R

```diff
@@ -0,0 +1,917 @@
+             "L244.SubsectorShrwt_bld_gcamusa",
+             "L244.SubsectorShrwtFllt_bld_gcamusa",
+             "L244.SubsectorInterp_bld_gcamusa",
+             "L244.SubsectorInterpTo_bld_gcamusa",
+             "L244.SubsectorLogit_bld_gcamusa",
+             "L244.StubTech_bld_gcamusa",
+             "L244.StubTechCalInput_bld_gcamusa",
+             "L244.StubTechMarket_bld",
+             "L244.GlobalTechIntGainOutputRatio",
+             "L244.GlobalTechInterpTo_bld",
+             "L244.GlobalTechEff_bld",
+             "L244.GlobalTechShrwt_bld_gcamusa",
+             "L244.GlobalTechCost_bld_gcamusa",
+             "L244.GlobalTechSCurve_bld",
+             "L244.HDDCDD_A2_GFDL"))
+  } else if(command == driver.MAKE) {
+
+    # Silence package checks
+    GCM <- Scen <- base.building.size <- base.service <- calibrated.value <- comm <-
+    degree.days <- efficiency <- efficiency_tech1 <- efficiency_tech2 <- fuel <-
+    gcam.consumer <- grid_region <- half_life_new <- half_life_stock <- input.cost <-
+    input.ratio <- internal.gains.market.name <- internal.gains.output.ratio <-
+    internal.gains.scalar <- market.name <- minicam.energy.input <- multiplier <-
+    object <- pcFlsp_mm2 <- pcGDP <- pcflsp_mm2cap <- pop <- region <- resid <-
+    satiation.adder <- satiation.level <- sector <- sector.name <- service <- share <-
+    share.weight <- share_tech1 <- share_tech2 <- share_type <- state <- steepness_new <-
+    steepness_stock <- stockavg <- subsector <- subsector.name <- supplysector <-
+    tech_type <- technology <- technology1 <- technology2 <-
+    thermal.building.service.input <- to.value <- value <- year <- year.fillout <- NULL
+
+    all_data <- list(...)[[1]]
+
+    # Load required inputs
+    A44.gcam_consumer_en <- get_data(all_data, "energy/A44.gcam_consumer")
+    A44.sector_en <- get_data(all_data, "energy/A44.sector")
+    calibrated_techs_bld_usa <- get_data(all_data, "gcam-usa/calibrated_techs_bld_usa")
+    states_subregions <- get_data(all_data, "gcam-usa/states_subregions")
+    A44.bld_shell_conductance <- get_data(all_data, "gcam-usa/A44.bld_shell_conductance")
+    A44.demandFn_flsp <- get_data(all_data, "gcam-usa/A44.demandFn_flsp")
+    A44.demandFn_serv <- get_data(all_data, "gcam-usa/A44.demandFn_serv")
+    A44.gcam_consumer <- get_data(all_data, "gcam-usa/A44.gcam_consumer")
+    A44.satiation_flsp <- get_data(all_data, "gcam-usa/A44.satiation_flsp")
+    A44.sector <- get_data(all_data, "gcam-usa/A44.sector")
+    A44.subsector_interp <- get_data(all_data, "gcam-usa/A44.subsector_interp")
+    A44.subsector_logit <- get_data(all_data, "gcam-usa/A44.subsector_logit")
+    A44.subsector_shrwt <- get_data(all_data, "gcam-usa/A44.subsector_shrwt")
+    A44.globaltech_cost <- get_data(all_data, "gcam-usa/A44.globaltech_cost")
+    A44.globaltech_eff <- get_data(all_data, "gcam-usa/A44.globaltech_eff") %>%
+      gather(year, value, matches(YEAR_PATTERN)) %>%
+      mutate(year = as.integer(year))
```

Comment:
Replace these new lines with call to new `gather_year` function (merged in #821 )